src/components/Interpretations/common/Interpretation/Interpretation.js

```diff
@@ -38,7 +47,8 @@ export const Interpretation = ({
             created={interpretation.created}
             id={interpretation.id}
             username={interpretation.user.displayName}
-            onClick={disabled ? undefined : onClick}
+            // XXX showSharingDialog is needed here to avoid the interpretations view to open when the SharingDialog is closed
```

Comment:
I think it's safe to remove this comment. The code is pretty self explanatory.

Reply:
I don't see how the comment is wrong, which makes me think it is needed then 😅